schema/derive/src/lib.rs

```diff
@@ -1,54 +1,36 @@
 //! Crate with derive `IntoSchema` macro
 
 #![allow(clippy::arithmetic_side_effects)]
-
-use proc_macro::TokenStream;
-use proc_macro2::{Ident, Span, TokenStream as TokenStream2};
-use quote::quote;
-use syn::{
-    parse::{Parse, ParseStream, Result},
-    parse_macro_input, parse_quote,
-    spanned::Spanned,
-    Attribute, Data, DataEnum, DataStruct, DeriveInput, Expr, Field, Fields, FieldsNamed,
-    FieldsUnnamed, GenericParam, Generics, LitStr, Meta, NestedMeta, Type,
-};
+// darling-generated code triggers this lint
+#![allow(clippy::option_if_let_else)]
```

Comment:
@mversic and I believe that we should stop enforcing the functional style project-wide. It should be reserved for crates that users are likely to touch, and crates which are already written in a functional style for consistency. We should remember to remove those.